lua/toggleterm/terminal.lua

```diff
@@ -586,7 +586,7 @@ if _G.IS_TEST then
     end
   end
 
-  M.__next_id = next_id
```

Comment:
I actually think this isn't necessary anymore. Since I wasn't exporting `next_id` before this was a way of trying to do it privately so I could use it in tests but now it's public this line can be removed and the corresponding usages of `M.__next_id` can just be `M.next_id`